index.html

```diff
@@ -11,61 +11,93 @@
     <title>Portfolio</title>
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="./main.css">
 
+
 </head>
 
 <body>
-    <header class="header">
-        <a class="logo">My logo</a>
-        <nav class="nav"><img src="./images/hamburger.png" alt="menu button"></nav>
-        <ul class="nav-list">
-            <li class="heading-list">Portifolio</li>
-            <li class="heading-list">About</li>
-            <li class="heading-list">Contact</li>
+
+    <nav>
+
+        <h1 class="logo">My Logo</h1>
+
+        <div class="openMenu"><i class="fa fa-bars"></i></div>
+        <ul class="mainMenu">
+            <li><a href="#section_1">Portfolio</a></li>
+            <li><a href="#section_2">About</a></li>
+            <li><a href="#section_3">Contact</a></li>
```

Comment:
The font stylings mostly match, but you have ignored the ever-important `line-height`.

Therefore:

- [ ] Please, make all these elements use `line-height: 44px` so your project looks like the Figma.

![image](https://user-images.githubusercontent.com/57421823/161737263-af69fff2-d458-4099-8b0b-6e98f57bd65c.png)

![image](https://user-images.githubusercontent.com/57421823/161737770-20aa6d87-bdcc-42f6-a1d9-62ce358d7363.png)


This will be the result!!! 💪